terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,55 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                = "search" # this must match the stack name the service deploys into
```

Comment:
Is this correct? The other service already live in this cluster ('search-web') has a 'stack_name' of 'search-service' - https://github.com/companieshouse/search.web.ch.gov.uk/blob/a4f855222f4ec0b11ff2a7cecb18b55aa251d788/terraform/groups/ecs-service/locals.tf#L3

Reply:
I had it as `search-service` before but was told to remove service as something similar was done for extensions. If `search-web` has already gone live with it perhaps that is the correct way of doing it then, can we confirm with someone?

Reply:
@dlloyd1-ch is correct. It is `search-service` in this case

Reply:
Done